Workaround against automatically generated setters/getters for public field.

Part of commit 5a60179 of upstream repository

--- bindings/yarp.i.orig	2013-08-30 17:38:46.678444665 +0200
+++ bindings/yarp.i	2013-08-30 17:39:13.098444002 +0200
@@ -86,6 +86,7 @@
 %ignore *::setKp(double);
 %ignore *::setKi(double);
 %ignore *::setKd(double);
+%ignore *::setKff(double);
 %ignore *::setScale(double);
 %ignore *::setOffset(double);
 %rename(attach_rpc_server) *::attach(yarp::os::RpcServer&);
